gcp_variant_transforms/testing/integration/run_tests_common.py

```diff
@@ -41,42 +41,57 @@ class TestRunner(object):
   """Runs the tests using pipelines API."""
 
   def __init__(self, tests):
-    # type: (List[Union[PreprocessorTestCase, VcfToBQTestCase]]) -> None
+    # type: (List[List[Union[PreprocessorTestCase, VcfToBQTestCase]]]) -> None
```

Comment:
As discussed offline: consider making a TestCaseInterface base class that these inherit from. That way, the interface of the test case class is clear (i.e. validate_result method should be implemented in all test case classes).